app/lang/en.json

```diff
@@ -219,7 +219,9 @@
     "selectCheckedLpa": "Select that you have checked your LPA and don't wish to make changes",
     "selectHappyToShareLpa": "Select that you are happy to share your LPA with your certificate provider",
     "checkYourLpa": "Check your LPA",
-    "youveNowFinishedFillingInTheLpa": "You’ve now finished filling in the LPA. Before you continue, carefully check all the answers.",
+    "whatIfIWantToMakeChanges": "What if I want to make a change to the LPA?",
+    "whatIfIWantToMakeChangesContent": "<p></p>",
```

Comment:
just want to check that you want this to be empty

Reply:
Yeah, the content on this page has an empty drop-down so I've left it blank for now.

Reply:
Ah actually, as this change was intended for `read-your-lpa` rather than `check-your-lpa` we already have some content in the details box there so I'll leave as it and drop the extra trans keys.